### ISSUE TITLE
Hi , What should I do if I want to train the model myself on conll2012 dataset?

### ISSUE THREAD

**Issue by sev777** [Riccorl/transformer-srl#12]

**Comment by Riccorl** [comment]
Do you have the conll-2012?

**Comment by sev777** [comment]
yes, I got it using "https://github.com/iesl/conll2012-preprocess-parsing", but the  format is different from transformer-srl

**Comment by sev777** [comment]
Hi, could provide a example format file for "conll-formatted-ontonotes-5.0-subset", which is used in  "train_test.sh".

**Comment by sev777** [comment]
Hi, could provide a example format file for "conll-formatted-ontonotes-5.0-subset", which is used in  "train_test.sh".
